RAGchain/preprocess/loader/deepdoctection_loader.py

```diff
@@ -37,11 +45,12 @@ def lazy_load(self, *args, **kwargs) -> Iterator[Document]:
         extracted_pages = self.extract_pages(result)
         for extracted_page in extracted_pages:
             if 'Table' in extracted_page:
-                yield Document(page_content=extracted_page['Table'], emetadata=extracted_page['Page_number'])
+                yield Document(page_content=extracted_page['Table'],
+                               metadata=({'Page_number': extracted_page['PageNumber']}))
```

Comment:
It will be awesome if table Document have metadata for recognizing that it is table.